src/query-commands.ts

```diff
@@ -34,16 +34,20 @@ const queryCreate = async (params: CmdParams) => {
     if (!ask) {
         ask = params.input;
     }
-
-    let result = await WAII.Query.generate({ ask: ask, dialect: dialect });
-    switch (params.opts['format']) {
+    if(!ask) {
+      console.log('Query generation requires a ask or tweak.')
+      console.error('Usage: waii query create <ask> <values> <flags>');
```

Comment:
What's <values> here? 

I think you can give one example for the error msg. like waii query create "how many tables in the database .. "

Reply:
we should exit, or throw exception instead of just printing error msg. (to avoid rest of the logic being executed).